Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBar.xaml

```diff
@@ -0,0 +1,290 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="ms-appx:///Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBarItem.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+
+    <DataTemplate x:Key="NormalTabTemplate">
+        <NavigationViewItem Content="{Binding Header}" />
+    </DataTemplate>
+    <DataTemplate x:Key="ContextualTabTemplate">
+        <NavigationViewItem Content="{Binding Header}">
+            <NavigationViewItem.Resources>
+                <SolidColorBrush x:Key="TopNavigationViewItemForeground" Color="{ThemeResource SystemAccentColor}" />
+                <SolidColorBrush x:Key="TopNavigationViewItemForegroundSelected" Color="{ThemeResource SystemAccentColor}" />
+                <SolidColorBrush x:Key="TopNavigationViewItemForegroundPointerOver" Color="{ThemeResource SystemAccentColorLight2}" />
+                <SolidColorBrush x:Key="TopNavigationViewItemForegroundPressed" Color="{ThemeResource SystemAccentColorLight2}" />
+            </NavigationViewItem.Resources>
+        </NavigationViewItem>
+    </DataTemplate>
+    <controls:TabbedCommandBarItemTemplateSelector x:Key="DefaultTabbedCommandBarItemTemplateSelector"
+                                                   Contextual="{StaticResource ContextualTabTemplate}"
+                                                   Normal="{StaticResource NormalTabTemplate}" />
+
+    <StaticResource x:Key="TabbedCommandBarContentBackground" ResourceKey="SystemControlChromeLowAcrylicWindowBrush" />
+    <Style TargetType="controls:TabbedCommandBar">
```

Comment:
Please follow the suggested guidance we use elsewhere of naming this and setting the default separately (and also splitting out the control template). See examples in things like the TokenizingTextBox.

Reply:
I think this is fixed in the latest commit. I'll double-cehc